lib/mimemagic/tables.rb

```diff
@@ -1,2066 +1,2066 @@
-    'wp6' => 'application/vnd.wordperfect',
-    'wpd' => 'application/vnd.wordperfect',
-    'wpg' => 'application/x-wpg',
-    'wpl' => 'application/vnd.ms-wpl',
-    'wpp' => 'application/vnd.wordperfect',
-    'wps' => 'application/vnd.ms-works',
-    'wri' => 'application/x-mswrite',
-    'wrl' => 'model/vrml',
-    'ws' => 'application/x-wonderswan-rom',
-    'wsc' => 'application/x-wonderswan-color-rom',
-    'wsgi' => 'text/x-python',
-    'wv' => 'audio/x-wavpack',
-    'wvc' => 'audio/x-wavpack-correction',
-    'wvp' => 'audio/x-wavpack',
-    'wvx' => 'audio/x-ms-asx',
-    'wwf' => 'application/x-wwf',
-    'x3f' => 'image/x-sigma-x3f',
-    'xac' => 'application/x-gnucash',
-    'xar' => 'application/x-xar',
-    'xbel' => 'application/x-xbel',
-    'xbl' => 'application/xml',
-    'xbm' => 'image/x-xbitmap',
-    'xcf' => 'image/x-xcf',
-    'xcf.bz2' => 'image/x-compressed-xcf',
-    'xcf.gz' => 'image/x-compressed-xcf',
-    'xdgapp' => 'application/vnd.flatpak',
-    'xhe' => 'audio/usac',
-    'xht' => 'application/xhtml+xml',
-    'xhtml' => 'application/xhtml+xml',
-    'xi' => 'audio/x-xi',
-    'xla' => 'application/vnd.ms-excel',
-    'xlam' => 'application/vnd.ms-excel.addin.macroEnabled.12',
-    'xlc' => 'application/vnd.ms-excel',
-    'xld' => 'application/vnd.ms-excel',
-    'xlf' => 'application/xliff+xml',
-    'xliff' => 'application/xliff+xml',
-    'xll' => 'application/vnd.ms-excel',
-    'xlm' => 'application/vnd.ms-excel',
-    'xlr' => 'application/vnd.ms-works',
-    'xls' => 'application/vnd.ms-excel',
-    'xlsb' => 'application/vnd.ms-excel.sheet.binary.macroEnabled.12',
-    'xlsm' => 'application/vnd.ms-excel.sheet.macroEnabled.12',
-    'xlsx' => 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
-    'xlt' => 'application/vnd.ms-excel',
-    'xltm' => 'application/vnd.ms-excel.template.macroEnabled.12',
-    'xltx' => 'application/vnd.openxmlformats-officedocument.spreadsheetml.template',
-    'xlw' => 'application/vnd.ms-excel',
-    'xm' => 'audio/x-xm',
-    'xmf' => 'audio/x-xmf',
-    'xmi' => 'text/x-xmi',
-    'xml' => 'application/xml',
-    'xpi' => 'application/x-xpinstall',
-    'xpm' => 'image/x-xpixmap',
-    'xps' => 'application/oxps',
-    'xsd' => 'application/xml',
-    'xsl' => 'application/xslt+xml',
-    'xslfo' => 'text/x-xslfo',
-    'xslt' => 'application/xslt+xml',
-    'xspf' => 'application/xspf+xml',
-    'xul' => 'application/vnd.mozilla.xul+xml',
-    'xwd' => 'image/x-xwindowdump',
-    'xz' => 'application/x-xz',
-    'yaml' => 'application/x-yaml',
-    'yml' => 'application/x-yaml',
-    'yt' => 'application/vnd.youtube.yt',
-    'z' => 'application/x-compress',
-    'z64' => 'application/x-n64-rom',
-    'zabw' => 'application/x-abiword',
-    'zip' => 'application/zip',
-    'zoo' => 'application/x-zoo',
-    'zsav' => 'application/x-spss-sav',
-    'zz' => 'application/zlib',
+    '123' => 'application/vnd.lotus-1-2-3'.freeze,
```

Comment:
Is it possible to remove all these free calls?

Reply:
I'll remove from this PR but I'm gonna dig into it more - I did this last year and I feel like I did it for a specific reason but I can't remember why anymore.